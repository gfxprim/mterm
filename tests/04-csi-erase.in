5 4
test!test\e[1A\e[K
