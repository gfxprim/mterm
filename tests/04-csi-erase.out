 -----
|test |
|test |
|     |
|     |
 -----
size 4x5 cursor 0x4
